website/docs/cookbooks/emacs.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Scala CLI within Emacs
+sidebar_position: 2
```

Comment:
```suggestion
sidebar_position: 14
```
Please make sure the position in the sidebar matches the order of cookbooks listed in `introduction.md`